### PR TITLE
Block all keydown when overlay

### LINKED ISSUE #1600: I can interact with the code editor when there is an overlay
**Describe the bug**
I can interact with the code editor when there is an overlay

**To Reproduce**
1. Upload a big file using drag and drop while looking at a model
2. use the keyboard to interact with the code editor.

**Expected behavior**
The overlay should block all user interactions.

**Screenshots**
https://user-images.githubusercontent.com/5587788/212120394-51784cb2-a2a9-45b9-a899-06a76141960e.mp4

### PR DESCRIPTION
**Pull request by djbarnwal**
Closes #1600

### CODE REVIEW

File: web-common/src/components/overlay/Overlay.svelte [hunk 1]
```diff
@@ -9,10 +9,8 @@
   // We'll need a better solution than this!
   function captureKeydown(event) {
     // capture all events
-    if (event.code === "Tab" || event.code === "Space") {
-      document.activeElement.blur();
-      event.preventDefault();
-    }
+    document.activeElement.blur();
```
Comment (hamilton) [inline web-common/src/components/overlay/Overlay.svelte:12]: does this block anything on top of the overlay that might take input?

Comment (djbarnwal) [reply web-common/src/components/overlay/Overlay.svelte:12]: Didn't get you. Isn't the overlay on top of everything else?

Comment (hamilton) [reply web-common/src/components/overlay/Overlay.svelte:12]: oh actually, nevermind, there are two kinds of overlays in our repository ~ this one, and the basis for a Modal. These function differently and will need to be handled separately.

**Review by hamilton — APPROVED** [review]
lgtm, thanks!

**Pull request merged**